setup.py

```diff
@@ -28,6 +28,7 @@
     "flake8>=5.0.0",
     "time_machine",
     "pyproj>=3.2.0",  # Pyproj is an optional, best-effort runtime dependency
+    "re-assert",
```

Comment:
we already have dirty_equals for these kind of things, wee should not add yet another utility here